source/docs/software/pathplanning/index.rst

```diff
@@ -1,7 +1,7 @@
 Path Planning
 =============
 
- Path Planning is the process of creating and following paths that your robot can autonomously follow. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, developing with the trajectory tracking API and visualizing your paths with PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for additional information about the API and non-commandbased usage.
+Path Planning is the process of creating and following paths or trajectories. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, trajectory following with ramsete and usage of PathWeaver.. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for additional information about the API and non-commandbased usage.
```

Comment:
Can we just say trajectories rather than paths or trajectories.  There really isn't any option to "follow" a path right?

You used ramsete again and I don't think we should use that alone.

2 periods after PathWeaver